python/nano/example/pytorch/quantization/inc/resnet18_cifar.py

```diff
@@ -158,4 +163,44 @@ def configure_optimizers(self):
 )
 
 trainer.fit(model, cifar10_dm)
-trainer.test(model, datamodule=cifar10_dm)
+
+fp32_model = model
+# FP32 testing
+start = time()
+outputs = trainer.test(fp32_model, datamodule=cifar10_dm)
+time1 = time() - start
+log_dict = outputs[0]
+print("FP32 model size: %.3f MB" % fp32_model.model_size)
+print("Throughput: %d s" % time1)
+print("Accuracy: top1=%.2f%%" % (log_dict['test_acc'] * 100))
+
+# Save FP32 checkpoint
+trainer.save_checkpoint('ResNet18_on_CIFAR_FP32_CKPT.pth')
+
+# Run post-training quantization
+int8_model = trainer.quantize(fp32_model,
+                              calib_dataloader=cifar10_dm.train_dataloader(),
+                              val_dataloader=cifar10_dm.test_dataloader(),
+                              metric=Accuracy(num_classes=10),
+                              accuracy_criterion={'higher_is_better': True, 'relative': 0.1},
+                              raw_return=True)
+
+# compile quantized model as pytorch-lightning
+int8_model = trainer.compile(int8_model, loss=criterion, metrics=[accuracy])
```

Comment:
if you set `raw_return=False`(default setting), then you will get a pl model directly. Still the pl model does not support `trainer.test` yet, since it will call `model.eval()` internally. We may support it in future.

Reply:
Still, this is very unnatural for the user; I think we should support `eval()` by default - a quantized model should use int8 in inference by default.

Reply:
yeah, I can change the default model.eval() to be model.eval(quantize=True), that may support the trainer.test

Reply:
I'll keep current `trainer.compile`. The `trainer.test` is not working on quantized pl model right now.